.github/workflows/as-docker-build.yml

```diff
@@ -16,6 +16,6 @@ on:
 
 jobs:
   check_as_image_build:
-    if: github.event_name == 'pull_request' || github.event_name == 'push'
+    if: github.event_name != 'push'
```

Comment:
why don't we remove `push.branches` form the workflow instead?

Reply:
I guess it would have the same effect. Let's give it a try! Thanks for the comment. 😉